src/classes/BDCBidOutcome.cls

```diff
@@ -1,4 +1,4 @@
-public class BDC_BidOutcome {
+public class BDC_Bid_Outcome {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.